TRANSLATING.md

```diff
@@ -1,42 +1,39 @@
 Translating Hedy
-======================
+================
 
-Hedy is now available in Dutch, Hindi, Bengali, Hungarian, Czech, French, English, Portugese, Brazilian Portugese, Greek, Mandarin, Hungarian and Spanish, but we'd love to support more languages!
+Hedy is now (partly) available in Arabic, Bengali, Bulgarian, Czech, Dutch, English, French, Frisian, German, Greek, Hindi, Hungarian, Indonesian, Italian, Mandarin, Portugese (Brazilian and Portugese), Spanish and Swahili but we'd love to support more languages! Text that is not yet translated will be shown in English.
 
 Help Hedy with translations (easy, no programming needed!)
-------------
+----------------------------------------------------------
 
-The easiest way to translate Hedy is by using or translation UI website!
+The easiest way to translate Hedy is by using the translation website of Weblate. You can find it [here](https://hosted.weblate.org/projects/hedy).
 
-Simply go to https://www.hedycode.com/translate/en/new and translate our texts that are shown on the left in the boxes on the right. When you are done, you can use the three download button at the end of the page, and [send us the files](mailto:hedy@felienne.com).
+## Older method (little harder and incomplete)
+
+Until recently this was the easy way, but harder to maintain for us. Translators allready using this method best finish their work en then move to the Weblate UI mentioned above.
```

Comment:
I am totally ok with removing this from the docs. We can leave the page available for a bit for the Hindi translators but otherwise let's not promote it.

Reply:
OK, I'll create a new version without reference to the older and publish it in this PR